plugins/home/src/components/CustomHomepage/CustomHomepageGrid.tsx

```diff
@@ -410,7 +410,9 @@ export const CustomHomepageGrid = (props: CustomHomepageGridProps) => {
           props.breakpoints ? props.breakpoints : theme.breakpoints.values
         }
         cols={
-          props.cols ? props.cols : { lg: 12, md: 10, sm: 6, xs: 4, xxs: 2 }
+          props.cols
+            ? props.cols
+            : { xl: 12, lg: 12, md: 10, sm: 6, xs: 4, xxs: 2 }
         }
         rowHeight={props.rowHeight ?? 60}
         onLayoutChange={handleLayoutChange}
```

Comment:
I think the layouts (one line lower) should also have an entry for xl, or should be replaced with xl? for me overwriting the cols config via props does not fully solve the issue.

Reply:
True, it should be replaced. I will fix this.